pyndustric/__main__.py

```diff
@@ -1,52 +1,57 @@
 import argparse
+
 import pyndustric
 import sys
 import time
 
 
-def parse_args():
+def create_args():
     parser = argparse.ArgumentParser(
         description="A compiler from Python to Mindustry's assembly (logic programming language)."
     )
 
-    arg = parser.add_argument
-    arg("files", help="source .py files to compile")
-    arg(
+    parser.add_argument("-file", "-f", action="store", type=str, help="source file to compile", required=True)
+    parser.add_argument(
         "-c",
         "--clipboard",
         action="store_true",
         help="copy the generated code to clipboard (requires `autoit`)",
     )
 
-    return parser.parse_args()
+    return parser
 
 
 def main():
-    args = parse_args()
+    parser = create_args()
+    args = parser.parse_args()
     complete_masm = ""
-    for file in args.files:
-        print(f"# reading {file}...", file=sys.stderr)
-        if file == "-":
-            source = sys.stdin.read()
-        else:
-            with open(file, encoding="utf-8") as fd:
-                source = fd.read()
-
-        print(f"# compiling {file}...", file=sys.stderr)
-        start = time.time()
-        masm = pyndustric.Compiler().compile(source)
-        complete_masm += masm
-        took = time.time() - start
-        print(masm)
-        print(
-            f"# compiled {file} with pyndustric {pyndustric.__version__} in {took:.2f}s",
-            file=sys.stderr,
-        )
+    # for file in args.files:
+    file = args.file
+    print(f"# reading {file}...", file=sys.stderr)
+    if file == "-":
+        source = sys.stdin.read()
+    else:
+        with open(file, encoding="utf-8") as fd:
+            source = fd.read()
+
+    print(f"# compiling {file}...", file=sys.stderr)
+    start = time.time()
+    masm = pyndustric.Compiler().compile(source)
+    complete_masm += masm
+    took = time.time() - start
+    print(masm)
+    print(
+        f"# compiled {file} with pyndustric {pyndustric.__version__} in {took:.2f}s",
+        file=sys.stderr,
+    )
 
     if args.clipboard:
-        import ait
+        try:
+            import ait
+            ait.copy(complete_masm)
+        except:
+            print("Require python module: autoit")
```

Comment:
While this is a nice addition, it's not related to fixing the issue with files, so I would prefer if it was a separate PR.